apps/bfd-data-fda/Jenkinsfile

```diff
@@ -135,7 +135,7 @@ mvn deploy "-DaltDeploymentRepository=bfd-mgmt-bfd-mgmt::default::https://bfd-mg
                                 awsAuth.assumeRole()
```

Comment:
should be able to add the exec:java stuff to the mvn build for fda-drug project and then we only need to run mvn deploy (test it out to make sure though!)

Reply:
done